localization/pose_initializer/launch/pose_initializer.launch.xml

```diff
@@ -7,6 +7,7 @@
   <arg name="stop_check_enabled"/>
 
   <arg name="sub_gnss_pose_cov" default="sub_gnss_pose_cov"/>
+  <arg name="fit_target" default="pointcloud_map"/>
```

Comment:
This may be a subject I want to hear from others, but the argument name `fit_target` is a bit ambiguous for me and may conflict with other components in the future. Besides, an argument starting with a verb can be misunderstood as if it were a true/false variable. So I prefer `fitter_target_map` or `height_fitter_target_map` rather than `fit_target` but what do you think?
(Variable names in the cpp code doesn't have to change I think.)

Reply:
Since this node will be used as part of another node, I think it's a good to add a namespace.
```
declare_parameter<std::string>("map_height_fitter.target");
declare_parameter<std::string>("map_height_fitter.map_loader_name");
```

And it would be better to give more detailed names to the top-level launch arguments, such as:
- `rviz_initial_pose_auto_fix_target` (for map_height_fitter param in rviz_adaptor)
- `gnss_initial_pose_auto_fix_target` (for map_height_fitter param in pose_initializer)

Reply:
Thank you for comments
Corrected according to your instructions
https://github.com/autowarefoundation/autoware.universe/pull/6340/commits/d52569cc2e9aefa3cadae80a84806c3c377d6662